doc/changelog/04-tactics/16868-zify-term.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:** :cmd:`Add Zify` now accepts terms instead of references.
+  This is typically useful to do ``Add Zify InjTyp (inj_thing x)`` in
+  a section where ``x`` is a section variable without having to make
+  an intermediate definition for ``inj_thing`` (`#16868
```

Comment:
```suggestion
  This lets you avoid creating intermediate definitions for terms
  that include section variables (such as ``x`` in ``(inj_thing x)``).
  Now you can use ``Add Zify InjTyp (inj_thing x)`` instead (`#16868
```

WDYT?  A bit more readable if the benefit appears at the beginning of the sentence so the reader immediately knows why they would care.